README.md

```diff
@@ -35,10 +35,23 @@ Please see [Learning Resources](#resources) for additional references.
 
 The CiscoNodeUtils gem provides utilities for management of Cisco network
 nodes. It is designed to work with Puppet and Chef as well as other
-open source management tools. This release supports Cisco NX-OS nodes
-running NX-OS 7.0(3)I2(1) and later.
+open source management tools.
 
-Please note: A virtual Nexus N9000/N3000 may be helpful for development and testing. Users with a valid [cisco.com](http://cisco.com) user ID can obtain a copy of a virtual Nexus N9000/N3000 by sending their [cisco.com](http://cisco.com) user ID in an email to <get-n9kv@cisco.com>. If you do not have a [cisco.com](http://cisco.com) user ID please register for one at [https://tools.cisco.com/IDREG/guestRegistration](https://tools.cisco.com/IDREG/guestRegistration)
+This CiscoNodeUtils gem release supports the following:
+
+Platform | OS | OS Version |
+---------|----|------------|
```

Comment:
if Markdown permits it might be nice to align the table in ASCII too:

```markdown
Platform         | OS    | OS Version           |
-----------------|-------|----------------------|
Cisco Nexus 30xx | NX-OS | 7.0(3)I2(1) and later
```